app/src/main/java/com/github/piasy/biv/example/App.java

```diff
@@ -86,15 +85,4 @@ static String getSystemProperty(String propName) {
         }
         return line;
     }
-
-    @Override
```

Comment:
why this?

Reply:
The newer version of LeakCanary apparently don't require to do that anymore, those function are deprecated.
I moved to the newer version to cut down the number of methods, still it's over the limit by a bit, so I just enabled multi-dex.

Reply:
okay cool!

Reply:
Done